src/store-api/src/metadata.rs

```diff
@@ -622,6 +629,45 @@ impl RegionMetadataBuilder {
             }
         }
     }
+
+    /// Changes column fulltext option.
+    fn change_column_fulltext(&mut self, options: HashMap<String, String>) {
+        for column_meta in self.column_metadatas.iter_mut() {
+            let mut fulltext = if let Ok(Some(f)) = column_meta.column_schema.fulltext_options() {
+                f
+            } else {
+                FulltextOptions::default()
+            };
+
+            if let Some(analyzer) = options.get(COLUMN_FULLTEXT_OPT_KEY_ANALYZER) {
+                match analyzer.to_ascii_lowercase().as_str() {
+                    "english" => {
+                        fulltext.enable = true;
+                        fulltext.analyzer = FulltextAnalyzer::English;
+                    }
+                    "chinese" => {
+                        fulltext.enable = true;
+                        fulltext.analyzer = FulltextAnalyzer::Chinese;
+                    }
+                    _ => {}
+                }
+            }
+            if let Some(case_sensitive) = options.get(COLUMN_FULLTEXT_OPT_KEY_CASE_SENSITIVE) {
+                match case_sensitive.to_ascii_lowercase().as_str() {
+                    "true" => {
+                        fulltext.enable = true;
+                        fulltext.case_sensitive = true;
+                    }
+                    "false" => {
+                        fulltext.enable = true;
+                        fulltext.case_sensitive = false;
+                    }
+                    _ => {}
```

Comment:
It's better to throw an error here. What will happen if the user sets the invalid full-text options while creating a table? @zhongzc

Reply:
will return error

```rust
                    return FulltextInvalidOptionSnafu {
                        msg: format!("{analyzer}, expected: 'English' | 'Chinese'"),
                    }
                    .fail();
```